cluster-autoscaler/main.go

```diff
@@ -190,7 +190,7 @@ var (
 	awsUseStaticInstanceList           = flag.Bool("aws-use-static-instance-list", false, "Should CA fetch instance types in runtime or use a static list. AWS only")
 	concurrentGceRefreshes             = flag.Int("gce-concurrent-refreshes", 1, "Maximum number of concurrent refreshes per cloud object type.")
 	enableProfiling                    = flag.Bool("profiling", false, "Is debug/pprof endpoint enabled")
-	clusterAPICloudConfigAuthoritative = flag.Bool("clusterapi-cloud-config-authoritative", false, "Treat the cloud-config flag authoritatively (do not fallback to using kubeconfig flag). ClusterAPI only")
+	clusterAPICloudConfigAuthoritative = flag.Bool("clusterapi-cloud-config-authoritative", false, "Treat the cloud-config flag authoritatively (do not fallback to using kubeconfig flag). Cluster API only")
```

Comment:
we might want to bring this change in on a separate review as it will require an owner from a different part of the repo to approve.

Reply:
I'll just undo this change to simplify; it wasn't related to the rest of this PR, I just thought it should have a space, but it's far from crucial.

Reply:
yeah, no worries. i think it will just make the review easier for us.